api/app/Http/Controllers/ChildEntityController.php

```diff
@@ -224,7 +224,7 @@ public function deleteOne($id, $childId)
         $model = $this->findParentEntity($id);
         $childModel = $this->findOrFailChildEntity($childId, $model);
 
-        $childModel->delete();
+        $model->deleteChild($childModel, $this->relationName);
```

Comment:
@xiphiaz If the child model is deleted directly on the child object, there is no way for the parent to know that the child was deleted. So by deleting it via the parent we can track that event. As the child object isn't by itself having revisions, but is part of the parent's revisions.